sentinel.conf

```diff
@@ -71,6 +71,22 @@ sentinel monitor mymaster 127.0.0.1 6379 2
 #
 # sentinel auth-pass mymaster MySUPER--secret-0123passw0rd
 
+# sentinel config-command <master-name> <command-name>
+# sentinel slaveof-command <master-name> <command-name>
```

Comment:
I think `sentinel set` and storing it in the same place/manner as other per-pod options such as auth-ass, parallel-syncs, etc. would be a more consistent route to take here. It would also make it available via the output of the `sentinel master` command which would allow it to be audited and synchronized to another sentinel.